main.go

```diff
@@ -44,13 +47,25 @@ func main() {
 	flag.Parse()
 	beforeRef := "master"
 	afterRef := "HEAD"
+	if flagCl != nil {
+		clHead, parent, err := clHeadAndParent(*flagCl)
+		if err != nil {
+			log.Fatalf("failed to get CL %s information", *flagCl)
+		}
+		if parent != "" {
```

Comment:
I don't see how `parent == ""` can happen with the Go repo. No-one is going to create a new disconnected branch. Instead, if `parent == ""`, call `log.Fatal`.

Reply:
Fixed